FinniversKit/Sources/Components/BrazePromotionView/BrazePromotionViewModel.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+
+public struct BrazePromotionViewModel {
+    let title: String
+    let text: String?
+    let image: String?
+    let primaryButtonTitle: String?
+    let dismissible: Bool?
+    let cardStyle: String?
```

Comment:
This property doesn't seem to be used anywhere. Do we use it from the apps for tracking or something else?

Reply:
CardStyle is currently something we require from Braze but as of yet we only accept "default". I haven't used it because we only need one style now, but I thought it would make sense to add it already to avoid head-scratching when we do want to use it as the logic from the main app is already there. What do you think?

Reply:
If we should need it it should be an enum, at least. Comparing strings given from an external provider is not something I think we should do in FinniversKit, that should be up to the app to decide.

And since we don't use it right now I think we should remove it and implement it if/when we need it :)